packages/peregrine/lib/talons/Breadcrumbs/useBreadcrumbs.js

```diff
@@ -28,9 +28,13 @@ const getPath = (path, suffix) => {
  * }}
  */
 export const useBreadcrumbs = props => {
-    const { categoryId, query } = props;
+    const {
+        categoryId,
+        queries: { getBreadcrumbsQuery }
+    } = props;
 
-    const { data, loading, error } = useQuery(query, {
+    const { data, loading, error } = useQuery(getBreadcrumbsQuery, {
+        fetchPolicy: 'cache-and-network',
```

Comment:
I've been thinking about our use of the default fetch policy lately. `cache-first` is appropriate when your data will rarely change, or when you know that you will be clearing said data sooner than later. I feel like we should change our default policy to `cache-and-network` as we always want to display data as quickly as possible (cache) but we also want to make sure that data is updated incase something changed on the server (network).

Reply:
I feel like I actually tried that at one point but couldn't get it to take. There is definitely [a configuration option](https://www.apollographql.com/docs/react/api/core/ApolloClient/#example-defaultoptions-object) on the client itself to set this.